### ISSUE TITLE
The libft's subject was updated

### ISSUE THREAD

**Issue by beskaska** [QuentinPerez/Maintest#42]
[en.subject.pdf](https://github.com/QuentinPerez/Maintest/files/3797972/en.subject.pdf)

**Comment by QuentinPerez** [comment]
thank you @beskaska, but what do you mean ?
If there is any change in this new subject, don't hesitate to do a PR with this changes :)

**Comment by beskaska** [comment]
Yep.. I'm already attach the subject FYI. There are several changes, especially in tasks about list.

**Comment by QuentinPerez** [comment]
Could you do a PR with the changes for the new subject ?
